coderd/database/queries/externalauth.sql

```diff
@@ -40,13 +40,20 @@ UPDATE external_auth_links SET
     oauth_refresh_token = $6,
     oauth_refresh_token_key_id = $7,
     oauth_expiry = $8,
-	oauth_extra = $9
+	oauth_extra = $9,
```

Comment:
I think there's some tabs vs spaces stuff going on here. would you mind cleaning up while we're touching this file?

Reply:
I will make a PR to address this more wholistically.

https://github.com/coder/coder/pull/19344